ortho_seq_code/tests/test_orthoseqs.py

```diff
@@ -23,21 +23,63 @@ def output_file_templates(basename):
         f"{basename}_reg11i1.npy",
         f"{basename}_Pa1i1.npy",
         f"{basename}_P1D.npy",
-        f"{basename}_varP1D.npy",
+        f"{basename}_varP1D.npy", #end of building first order space
+
+    ]
+
+def output_file_2ndorder_templates(basename):
```

Comment:
if you use saving all the arrays as npz format you would not have to do this also you should be using refex to see if you have 2 in the string or something like that  if its 2nd order